docs/api-overview.mdx

```diff
@@ -197,7 +203,19 @@ Let's break this down to understand what is happening here:
 }
 ```
 
-> _¹The `FullNameCredential` also has a `MiddleNameCredential`, but it's `optional`._
+
+- `PhoneCredential` is a "[single attribute atomic credential](/schema#schema-definitions)" and therefore does not have any sub-credentials. Because we set a `allowUserInput` attribute of `true`, the user will be given an opportunity to enter a new value if they choose."
+
+```json title="PhoneCredential"
+{
+    "type": "PhoneCredential",
+    "issuers": [],
+    "mandatory": "if_available",
+    "allowUserInput": true
+}
+```
+
+Check that the `mandatory` is not present, the reason is when `allowUserInput`, the credential may be created and it is optional.
```

Comment:
```suggestion
Note that the `mandatory` is not present. This is thanks to `allowUserInput` being `true`, marking that the credential may be created and it is optional.
```

Reply:
Although I really do not like this implicit behavior. I would really prefer the `mandatory` field needs to be a particular value when `allowUserInput` is used.

Reply:
Why would `mandatory` need to be a particular value when `allowUserInput` is used? shouldn't the two be completely independent?

Reply:
I'm going to remove this part since `allowUserInput` is unrelated to mandatory.